packages/contracts-bedrock/deploy-config/sepolia.json

```diff
@@ -56,5 +56,20 @@
   "proofMaturityDelaySeconds": 604800,
   "disputeGameFinalityDelaySeconds": 302400,
   "respectedGameType": 0,
-  "useFaultProofs": true
+  "useFaultProofs": true,
+  "livenessModuleInterval": 31536000,
+  "livenessModuleThresholdPercentage": 20,
+  "livenessModuleMinOwners": 2,
+  "securityCouncilThreshold": 2,
```

Comment:
The 20/2/2 values make sense, but is 1 year interval what we plan to use in production and on sepolia? Where is this documented to verify?